controller_manager/src/controller_manager/controller_manager_interface.py

```diff
@@ -75,37 +75,28 @@ def unload_controller(name):
         return False
 
 def start_controller(name):
-    return start_stop_controllers([name], True)
+    return start_stop_controllers(start=[name])
 
 def start_controllers(names):
-    return start_stop_controllers(names, True)
+    return start_stop_controllers(start=names)
 
 def stop_controller(name):
-    return start_stop_controllers([name], False)
+    return start_stop_controllers(stop=[name])
 
 def stop_controllers(names):
-    return start_stop_controllers(names, False)
+    return start_stop_controllers(stop=names)
 
-def start_stop_controllers(names, st):
+def start_stop_controllers(start=[], stop=[]):
     rospy.wait_for_service('controller_manager/switch_controller')
     s = rospy.ServiceProxy('controller_manager/switch_controller', SwitchController)
-    start = []
-    stop = []
     strictness = SwitchControllerRequest.STRICT
-    if st:
-        start = names
-    else:
-        stop = names
     resp = s.call(SwitchControllerRequest(start, stop, strictness))
     if resp.ok == 1:
-        if st:
-            print "Started {} successfully".format(names)
-        else:
-            print "Stopped {} successfully".format(names)
+        if start:
+            print "Started {} successfully".format(start)
```

Comment:
Not really from this PR, but you could add parenthesis to `print`. Here and below

Reply:
@bmagyar Do you mean `print(msg)` style shouldn't be in kinetic?

Reply:
I think he wants to cherry-pick only that style/deprecation change into `kinetic-devel`.
This PR is targeting `modelodic-devel`, btw.